packages/astroport/src/factory.rs

```diff
@@ -50,6 +50,8 @@ pub struct PairConfig {
     /// Whether a pair type is disabled or not. If it is disabled, new pairs cannot be
     /// created, but existing ones can still read the pair configuration
     pub is_disabled: Option<bool>,
+    /// Setting to true will disable creating a pool in generator for this pair type
+    pub is_generator_disabled: Option<bool>,
```

Comment:
Do we need to use Option here? Maybe just use bool? Let's set both to bool by the way

Reply:
It makes sense. Changed and made the default parameter `false`